src/libsecutils/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# =============================
+# Define targets
+# =============================
+add_library(security-utilities_library SHARED)
+add_library(security-utilities::library ALIAS security-utilities_library)
+
+set_target_properties(security-utilities_library PROPERTIES
+    EXPORT_NAME library
+    OUTPUT_NAME secutils
+    SOVERSION ${security-utilities_VERSION_MAJOR}
+    VERSION ${security-utilities_VERSION_MAJOR}.${security-utilities_VERSION_MINOR}.${security-utilities_VERSION_PATCH}
+)
+
+target_compile_features(security-utilities_library PRIVATE c_std_90)
+set_target_properties(security-utilities_library PROPERTIES C_EXTENSIONS ON)
+
+add_subdirectory(include)
+add_subdirectory(src)
+
+target_link_libraries(security-utilities_library
+    PUBLIC
+    OpenSSL::SSL
+    OpenSSL::Crypto
+
+    $<$<BOOL:${SECURITY_UTILITIES_USE_UTA}>:-luta>
```

Comment:
This does not yet work properly.
It works of you replace in by `$<$<BOOL:${SECUTILS_USE_UTA}>:-luta>`
and place earlier in this file:
```
if(SECURITY_UTILITIES_USE_UTA OR DEFINED ENV{SECUTILS_USE_UTA})
    set(SECUTILS_USE_UTA ON)
endif()
```